account/views.py

```diff
@@ -1,4 +1,54 @@
-from django.shortcuts import render
-from django.views       import View
-from django.http        import JsonResponse
+
+import json
+import ast
+import httplib2
+import google.oauth2.credentials
+import jwt
+import requests
+
+from google.oauth2          import id_token
+from django.shortcuts       import render
+from django.views           import View
+from django.http            import JsonResponse
+
+from apiclient              import discovery
+from oauth2client           import client
+from pinterrorist.settings  import SECRET_KEY
+
+from .models                import User, Social, Follow
+
+class GoogleLoginView(View):
+    def post(self, request):
+        token       = request.headers["Authorization"]
+        url         = 'https://www.googleapis.com/oauth2/v1/tokeninfo?access_token='
+        response    = requests.get(url+token)
+        user        = response.json()
+        
+        try:
+            if User.objects.filter(social_id = user['user_id']).exists(): 
+
+                user_info           = User.objects.get(social_id=user['user_id']) 
+                encoded_jwt         = jwt.encode({'id': repr(user_info.id)}, SECRET_KEY, algorithm='HS256')
+
+                return JsonResponse({ 
+                    'access_token'  : encoded_jwt.decode('UTF-8'),
+                    "message"       : "ALREADY SIGNED-UP"
+                }, status = 200) 
+            else:
+                new_user_info = User( 
+                    social_id       = user['user_id'],
+                    social_service  = Social.objects.get(name = "google"),
+                    email           = user.get('email', None)
+                )
+                new_user_info.save() 
+                encoded_jwt         = jwt.encode({'id': repr(new_user_info.id)}, SECRET_KEY, algorithm='HS256')
+
+                return JsonResponse({
+                'access_token'      : encoded_jwt.decode('UTF-8'),
+                'username'          : new_user_info.social_id,
+                'user_pk'           : new_user_info.id,
+                "message"           : "NEW ACCOUNT CREATED"
```

Comment:
들여쓰기 및 작은따옴표 또는 큰따옴표로 통일 시켜주세요.
```suggestion
                    'access_token'      : encoded_jwt.decode('UTF-8'),
                    'username'          : new_user_info.social_id,
                    'user_pk'           : new_user_info.id,
                    "message"           : "NEW ACCOUNT CREATED"
```